.github/workflows/vib-publish.yaml

```diff
@@ -0,0 +1,113 @@
+      - master
+    paths:
+      - 'bitnami/wordpress/Chart.yaml'
+env:
+  CSP_API_URL: https://console.cloud.vmware.com
+  CSP_API_TOKEN: ${{ secrets.CSP_API_TOKEN }}
+  VIB_PUBLIC_URL: https://cp.bromelia.vmware.com
+jobs:
+  get-chart:
+    runs-on: ubuntu-latest
+    name: 'Get modified charts'
+    outputs:
+      chart: ${{ steps.get-chart.outputs.chart }}
+      result: ${{ steps.get-chart.outputs.result }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: charts
+          fetch-depth: 2 # to be able to obtain files changed in the latest commit
+      - id: get-chart
+        name: 'Get modified charts'
+        run: |
+          cd charts
+          files_changed="$(git show --pretty="" --name-only)"
+          # Adding || true to avoid "Process exited with code 1" errors
+          charts_dirs_changed="$(echo "$files_changed" | xargs dirname | grep -o "bitnami/[^/]*" | sort | uniq || true)"
+          # Using grep -c as a better alternative to wc -l when dealing with empty strings."
+          num_charts_changed="$(echo "$charts_dirs_changed" | grep -c "bitnami" || true)"
+          num_version_bumps="$(echo "$files_changed" | grep Chart.yaml | xargs git show | grep -c "+version" || true)"
+
+          if [[ "$num_charts_changed" -ne "$num_version_bumps" ]]; then
+            # Changes done in charts but version not bumped -> ERROR
+            echo "::set-output name=error::Detected changes in charts without version bump in Chart.yaml.\nCharts changed: ${num_charts_changed}\n${charts_dirs_changed}\nVersion bumps detected: ${num_version_bumps}"
+            echo "::set-output name=result::fail"
+          elif [[ "$num_charts_changed" -eq "1" ]]; then
+            # Changes done in only one chart -> OK
+            chart_name=$(echo "$charts_dirs_changed" | sed "s|bitnami/||g")
+            echo "::set-output name=chart::${chart_name}"
+            echo "::set-output name=result::ok"
+          else
+            # Changes done in more than chart -> FAIL
+            echo -e "::set-output name=error::Changes detected in more than one chart directory:\n${charts_dirs_changed}\nThe publish process will be stopped. Please create different commits for each chart."
+            echo "::set-output name=result::fail"
+          fi
+      - id: show-error
+        name: 'Show error'
+        if: ${{ steps.get-chart.outputs.result == 'fail' }}
+        uses: actions/github-script@v3
+        with:
+          script: |
+              core.setFailed('${{ steps.get-chart.outputs.error }}')
+
+  vib-publish:
+    runs-on: ubuntu-latest
+    needs: get-chart
+    if: ${{ needs.get-chart.outputs.result == 'ok' }}
+    environment: vmware-image-builder
+    name: Publish
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout Repository
+        with:
+          path: charts
+      - uses: vmware-labs/vmware-image-builder-action@main
+        name: Verify and publish ${{ needs.get-chart.outputs.chart }}
```

Comment:
I would say just `Publish`. It is like `mvn package`, it involves prior phases.

Reply:
I don't think it would be so obvious that a `Publish` pipeline would also verify, unlike `package` phase, which is indeed implicit.
In fact, this is our choice to verify everything again, but we could rely on the success of PR Verify to avoid verifying everything again.

Reply:
It was not obvious the first time when I used Maven 😄 but it makes sense.

Up to you, I don't have a strong thought here.